golang/publisherConnector/main.go

```diff
@@ -0,0 +1,354 @@
+								Value: "Model of a Temperature Sensor Twin",
+								Lang:  "en",
+							},
+						},
+					},
+					{
+						Key: helpers.PROPERTY_KEY_SPACE_NAME,
+						Value: &ioticsApi.Property_StringLiteralValue{
+							StringLiteralValue: &ioticsApi.StringLiteral{
+								Value: "Space A",
+							},
+						},
+					},
+					{
+						Key: helpers.PROPERTY_KEY_COLOR,
+						Value: &ioticsApi.Property_StringLiteralValue{
+							StringLiteralValue: &ioticsApi.StringLiteral{
+								Value: "#9aceff",
+							},
+						},
+					},
+					{
+						Key: helpers.PROPERTY_KEY_CREATED_BY,
+						Value: &ioticsApi.Property_StringLiteralValue{
+							StringLiteralValue: &ioticsApi.StringLiteral{
+								Value: "Lorenzo",
+							},
+						},
+					},
+					{
+						Key: helpers.PROPERTY_KEY_DEFINES,
+						Value: &ioticsApi.Property_UriValue{
+							UriValue: &ioticsApi.Uri{
+								Value: helpers.PROPERTY_VALUE_SAREF_TEMPERATURE_SENSOR,
+							},
+						},
+					},
+					{
+						Key: helpers.PROPERTY_KEY_SAREF_HAS_MODEL,
+						Value: &ioticsApi.Property_StringLiteralValue{
+							StringLiteralValue: &ioticsApi.StringLiteral{
+								Value: "SET-LATER",
+							},
+						},
+					},
+				},
+				Feeds: []*ioticsApi.UpsertFeedWithMeta{
+					{
+						Id: "temperature",
+						Properties: []*ioticsApi.Property{
+							{
+								Key: helpers.PROPERTY_KEY_LABEL,
+								Value: &ioticsApi.Property_LangLiteralValue{
+									LangLiteralValue: &ioticsApi.LangLiteral{
+										Value: "Temperature",
+										Lang:  "en",
+									},
+								},
+							},
+							{
+								Key: helpers.PROPERTY_KEY_COMMENT,
+								Value: &ioticsApi.Property_LangLiteralValue{
+									LangLiteralValue: &ioticsApi.LangLiteral{
+										Value: "Current temperature of a Room",
+										Lang:  "en",
+									},
+								},
+							},
+						},
+						Values: []*ioticsApi.Value{
+							{
+								Label:    "reading",
+								Comment:  "Temperature in Celsius degrees",
+								DataType: "integer",
+								Unit:     helpers.UNIT_DEGREE_CELSIUS,
+							},
+						},
+					},
+				},
+			},
+		},
+	)
+	if err != nil {
+		log.Fatalf("Could not Upsert Twin: %v", err)
+	}
+	log.Printf("Created Twin Model %s", twinTempModelIdentity.Did())
+
+	/*** CREATE 2 TWIN TEMPERATURE FROM THAT MODEL ***/
+	/* Let's copy-paste the entire Twin Model upsert info with the right changes:
+	   - twin_did
+	   - 1st Twin Property
+	   - Label
+	   - Comment
+	   - hostMetadataAllowList
+	   - hostAllowList.
+	   The Twins' setup phase ends here. */
+
+	// The following is mapping between Sensor ID and Twin DID (eventually we'll need the latter info to share data)
+	roomNumberToTwinId := make(map[int]string)
+
+	for roomNumber := 1; roomNumber <= 2; roomNumber++ {
+		twinTemperatureIdentity := helpers.CreateTwinWithControlDelegation(
+			resolverClient,
+			AGENT_SEED, fmt.Sprintf("Sensor%d", roomNumber),
+			agentIdentity,
+		)
+
+		_, err = apiContext.TwinAPI.UpsertTwin(
```

Comment:
I wonder if it might be prudent to add some very simple response==OK check to set good practices. 
Don't worry if you think it complicates it.

Reply:
I guess not, but it may be good to grab the twinId from the response, just to demonstrate its there etc.

Reply:
I've added a print of the response for the Twin Model's Upsert Twin, just to show what's inside